### ISSUE TITLE
Does anyone remember the "rm -rf" trick in Edxposed?

### ISSUE THREAD

**Issue by TheInterestingSoul** [QiushiWu/qiushiwu.github.io#17]
> [Edxposed](https://github.com/ElderDrivers/EdXposed) is an Android hook framework. On May 27, 2020, two persons made 3 PRs that contained `rm -rf /data`, then it has triggered the CI build and broke about 50 Android phones.

What happened on the Linux kernel this time reminds me of this mistake a year ago. Were there any similar and more well-known "work" before? I just want to make a collection. :watermelon:

**Comment by loneicewolf** [comment]
Yeah..  I following this to get notified  if such a collection comes up. :+1:

**Comment by evolvingfridge** [comment]
Here are few related:

[Facebooks Secret Mood Manipulation Research](https://www.theatlantic.com/technology/archive/2014/06/everything-we-know-about-facebooks-secret-mood-manipulation-experiment/373648/)

[Wikipedia is not a laboratory](https://en.wikipedia.org/wiki/Wikipedia:What_Wikipedia_is_not)
Wikipedia had multiple past incidents, but I did not search for them.

[Highly Relevant Court Test Conducted by Researchers](https://casetext.com/case/sandvig-v-barr)

**Comment by loneicewolf** [comment]
@evolvingfridge  Yeah! Especially those 2 last!  te backdoor attempts!  I mean  this is exactly it!  Why even try? This.. is stupid.
